src/app/interface-form/interface-form.component.ts

```diff
@@ -18,4 +18,8 @@ export class InterfaceFormComponent {
       Validators.max(this.userDeposit),
     ]),
   });
+
+  onSubmit(form: FormGroup) {
+    console.log(form);
```

Comment:
Don't forget to remove it

Reply:
I plan to leave this method for implementation in the next P.R.